### ISSUE TITLE
[Crash] Sulfuric acid causes crash on finding where it's used

### ISSUE THREAD

**Issue by jarquafelmu** [ElmarWueest/Recexplo#9]
Sulfuric acid caused the game to crash upon clicking the "find where it's used" radio option.

```
19417.313 Error MainLoop.cpp:1035: Exception at tick 28217682: Error while running event RecExplo::on_gui_checked_state_changed (ID 3)
Gui element with name product_table_coal already present in the parent element.
stack traceback:
	__RecExplo__/lua/explo_gui.lua:517: in function 'draw_product'
	__RecExplo__/lua/explo_gui.lua:491: in function 'draw_recipe'
	__RecExplo__/lua/explo_gui.lua:407: in function 'create_all_where_used'
	__RecExplo__/lua/explo_gui.lua:356: in function 'create_all_recipes_container'
	__RecExplo__/lua/explo_gui.lua:332: in function 'create_results'
	__RecExplo__/lua/explo_gui.lua:327: in function 'update_results'
	__RecExplo__/lua/explo_gui.lua:69: in function 'update'
	__RecExplo__/lua/events.lua:255: in function <__RecExplo__/lua/events.lua:240
19417.313 Error ServerMultiplayerManager.cpp:96: MultiplayerManager failed: "Error while running event RecExplo::on_gui_checked_state_changed (ID 3)
Gui element with name product_table_coal already present in the parent element.
stack traceback:
	__RecExplo__/lua/explo_gui.lua:517: in function 'draw_product'
	__RecExplo__/lua/explo_gui.lua:491: in function 'draw_recipe'
	__RecExplo__/lua/explo_gui.lua:407: in function 'create_all_where_used'
	__RecExplo__/lua/explo_gui.lua:356: in function 'create_all_recipes_container'
	__RecExplo__/lua/explo_gui.lua:332: in function 'create_results'
	__RecExplo__/lua/explo_gui.lua:327: in function 'update_results'
	__RecExplo__/lua/explo_gui.lua:69: in function 'update'
	__RecExplo__/lua/events.lua:255: in function <__RecExplo__/lua/events.lua:240>"
```

**Comment by ElmarWueest** [comment]
should be fixed with 0.16.7

**Comment by jarquafelmu** [comment]
Yay!
